.azure-pipelines/warden.yml

```diff
@@ -24,6 +24,34 @@ jobs:
       python setup.py sdist -d $(Build.ArtifactStagingDirectory) --format zip
     displayName: 'Build Package'
 
+  - task: CopyFiles@2
+    displayName: 'Copy files for github release'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)'
+      Contents: 'scripts/*/verify_agent_os.py'
+      TargetFolder: '$(Build.BinariesDirectory)/sdk-tools'
+      OverWrite: true
+      flattenFolders: true
+  
+  - task: ArchiveFiles@2
+    displayName: 'Archive files for GitHub release'
+    inputs:
+      rootFolderOrFile: '$(Build.BinariesDirectory)/sdk-tools'
+      includeRootFolder: false
+      archiveFile: '$(Build.BinariesDirectory)/sdk-tools.zip'
+
+  - task: GitHubRelease@0
+    displayName: 'GitHub release of public sdk tools'
+    condition: and(succeededOrFailed(), ne(variables['System.TeamProject'], 'public'))
+    inputs:
+      gitHubConnection: GithubReleases
```

Comment:
@weshaggard steps referencing connections like this one are disallowed from running in public builds. I don't think this will be an issue, but just a a policy are we ok with referencing service connections by name?

Reply:
Yes we should definitely avoid this in our public pipelines. I don't recall but will this fail if the connection isn't defined in our public pipeline/team?

Reply:
Yes, the task requires a github connection in the specific devOps project.

Reply:
Ok I think I'm going to suggest we don't put this step here then but instead create a release pipeline which will do this part based on the artifacts that we will publish from this build pipeline.

Reply:
I have created the release pipeline in the internal project and removed the task here, Can i go ahead and merge this?

Reply:
`azure-sdk-tools-warden to github` right?  I took a look. Looks good!